README.md

```diff
@@ -53,11 +63,40 @@ message Cell {
   }
 }
 ```
-See the `model` subproject for details.
+Definitely, we can use this model but it is so easy to make some mistake when 
+the model becomes invalid state. What can we do to avoid this?
+
+It would be much better if we could add the following:
+
+1. Mark some fields as `required` to admit that this is important
```

Comment:
"Mark some fields as `required` to state the absolute necessity to fill them with real values."